[2024-03-03T03:42:10.317+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_data_files.get_files_1 scheduled__2016-12-23T00:00:00+00:00 [queued]>
[2024-03-03T03:42:10.331+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_data_files.get_files_1 scheduled__2016-12-23T00:00:00+00:00 [queued]>
[2024-03-03T03:42:10.332+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-03T03:42:10.350+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): get_files_1> on 2016-12-23 00:00:00+00:00
[2024-03-03T03:42:10.358+0000] {standard_task_runner.py:60} INFO - Started process 4738 to run task
[2024-03-03T03:42:10.361+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'fetch_data_files', 'get_files_1', 'scheduled__2016-12-23T00:00:00+00:00', '--job-id', '2163', '--raw', '--subdir', 'DAGS_FOLDER/dummy_dag.py', '--cfg-path', '/tmp/tmp0d4i9t9e']
[2024-03-03T03:42:10.365+0000] {standard_task_runner.py:88} INFO - Job 2163: Subtask get_files_1
[2024-03-03T03:42:10.387+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.11/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-03T03:42:10.444+0000] {task_command.py:423} INFO - Running <TaskInstance: fetch_data_files.get_files_1 scheduled__2016-12-23T00:00:00+00:00 [running]> on host 716a53fa82e1
[2024-03-03T03:42:10.585+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_data_files' AIRFLOW_CTX_TASK_ID='get_files_1' AIRFLOW_CTX_EXECUTION_DATE='2016-12-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2016-12-23T00:00:00+00:00'
[2024-03-03T03:42:13.526+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-03-03T03:42:13.542+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=fetch_data_files, task_id=get_files_1, execution_date=20161223T000000, start_date=20240303T034210, end_date=20240303T034213
[2024-03-03T03:42:13.592+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-03T03:42:13.630+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
